packages/domain/lib/use_case/use_case.dart

```diff
@@ -1,5 +1,7 @@
-abstract class UseCase {
-  bool call(String text, int first, int last);
+part of 'palindrome_use_case.dart';
+
+abstract class UseCase<Input, Output> {
+  Output call(String text, int first, int last);
```

Comment:
in this case Input type is useless. create separate model, for ex. PalindromeParams

PalindromeParams
|-text
|-first
|-last

and pass it as input

Reply:
fixed